README.md

```diff
@@ -1365,6 +1366,22 @@ For monitoring of different deployments, a unique UUID for each package is avail
 }
 ```
 
+### Application Load Balancer Event Source
+
+Zappa can be used to handle events triggered by Application Load Balancers (ALB). This can be useful in a few circumstances:
+- Since API Gateway has a hard limit of 30 seconds before timing out, you can use an ALB for longer running requests.
+- API Gateway is billed per-request; therefore, costs can become excessive with high throughput services. ALBs pricing model makes much more sense financially if you're expecting a lot of traffic to your Lambda.
+- ALBs can be placed within a VPC, which may make more sense for private endpoints than using API Gateway's private model (using AWS PrivateLink).
+
+Zappa can't automatically provision an ALB for you, but you can set your Lambda to be an ALB target by doing the following:
```

Comment:
TODO: Update documentation around this, as we're pulling in the provisioning code as well.